deluge/tests/test_web_api.py

```diff
@@ -139,13 +139,13 @@ def get_host_status(self):
         self.assertEquals(status, tuple(status))
 
     def test_get_host(self):
-        self.assertEquals(self.deluge_web.web_api._get_host("invalid_id"), None)
+        self.assertFalse(self.deluge_web.web_api._get_host("invalid_id"))
```

Comment:
Well I did originally have it equal to `[]` but I thought since we test for `False` in code whenever there is an empty list it would be a more equivalent test...